### Given
Density of bacteria in plate = $\rho= 10/cm^2$
Area $A=10cm^2$
Probability of a bacteria carrying mutation = $\mu$


### Facts

Bacteria on plate are like 'white and black' balls in a box with thw white ones representing the mutated copies.
Thus, assuming total number $N \sim Poisson(\lambda)$ and the coordinates $(X_i,Y_i)$ to be in a uniform square(assume the 'dish' to be square ) it is safe to assume, that this is a poisson point process with intensity $\mu$ and the mutated 'whites' make it a poisson coloring problem.

Hence, if $MW$ represents the mutated whites and $UB$ represent the unmutated black balls, we have 

$MW \sim PPP(\mu\rho)$ and $UB \sim PPP((1-\mu)\rho)$


Thus for part (a): P(probability that n 'mutated whites' exist) = $\frac{e^{-\rho\mu}(\mu\rho)^n}{n!}$



### Part (b)

In [4]:
from math import log, exp
t_min_grow = log(10)/log(1.05)

In [5]:
print t_min_grow

47.1936328191


Radius of cell growing for last $t$ minutes: $1.05^t \times 10^{-3}cm$. Thus for a cell to form a detectable cluster it should grow for atleast: $\frac{log(10)}{log(1.05}=47.19$ minutes

And the total time for grow is $60$ minutes, so that MAX wait time before growth starts is $60-47.19=12.81$ minutes

In [7]:
print(1-exp(-12.81/20))

0.472971156268


$\delta = p(\text{wait time $\leq$ 12.81 minutes}) = 1-\exp(-12.81/20) = 0.473$


So now we need these criteria for calling a ball white: 
- It gets mutated (PPP)
- It has a waiting time of <12.81 minutes

So we have a new $PPP(\rho \mu \delta)$ and 
$\frac{17}{\rho \times A} = \exp(-\rho \mu \delta) \times \frac{(-\rho \mu \delta)^}{}